docs/application/native/guides/security/dpm.md

```diff
@@ -5,6 +5,9 @@ The Device Policy Management (DPM) framework supports enterprise applications by
 
 DPM consists of a device policy client library and a device policy manager. The device policy manager manages all device policies and provides interfaces for the device policy client library. The device policy client library contains the device administration functions the client application can call. Internally, the device policy client library communicates with the device policy manager using a built-in remote method invocation engine.
 
+[!Note]
```

Comment:
```suggestion
[!NOTE]
```

Reply:
I applied the suggestion. Thanks for the review.